hooks/hooks.go

```diff
@@ -11,15 +11,19 @@ const (
 	// None of these hooks are ever associated with a relation; each of them
 	// represents a change to the state of the unit as a whole. The values
 	// themselves are all valid hook names.
-	Install            Kind = "install"
-	Start              Kind = "start"
-	ConfigChanged      Kind = "config-changed"
-	UpgradeCharm       Kind = "upgrade-charm"
-	Stop               Kind = "stop"
-	ActionRequested    Kind = "action-requested" // TODO: remove in charm v5, DEPRECATED
-	Action             Kind = "action"
-	CollectMetrics     Kind = "collect-metrics"
-	MeterStatusChanged Kind = "meter-status-changed"
+	Install               Kind = "install"
+	Start                 Kind = "start"
+	ConfigChanged         Kind = "config-changed"
+	UpgradeCharm          Kind = "upgrade-charm"
+	Stop                  Kind = "stop"
+	ActionRequested       Kind = "action-requested" // TODO: remove in charm v5, DEPRECATED
+	Action                Kind = "action"
+	CollectMetrics        Kind = "collect-metrics"
+	UpdateStatus          Kind = "update-status"
```

Comment:
could this please be moved to the end of the list, as where it is now splits up the CollectMetrics and MeterStatusChanged hooks which belong together. Same comment applies to other files.